.travis.yml

```diff
@@ -6,7 +6,7 @@ python: 3.7.4
 jobs:
   include:
     - stage: lint
-      script: ansible-lint playbooks/*.yml
+      script: ansible-lint playbooks/**/*.yml roles/**/*.yml
```

Comment:
From my experience this is already done by `playbooks/*.yml`. Don't ask me why :grimacing:

Reply:
Look at this output from `odoo-provisioning` repo: https://gist.github.com/enricostano/d4822a28e7db2b70d5f0735005388c0a

Reply:
I have the hypothesis that including the role makes it read the roles code as the `ansible` command does but not all. Check yourself:

```
➜  timeoverflow-provisioning install-es-2.4.6 ✗ ansible-lint roles/**/*.yml                                                                                                                                                                        <<<
[206] Variables should have spaces before and after: {{ var_name }}
roles/elasticsearch/tasks/main.yml:13
  template: src=elasticsearch.j2 dest={{instance_sysd_script}} mode=0644 force=yes

[206] Variables should have spaces before and after: {{ var_name }}
roles/elasticsearch/vars/main.yml:3
instance_sysd_script: "{{sysd_script | dirname }}/{{es_instance_name}}_{{sysd_script | basename}}"
```

```
➜  timeoverflow-provisioning install-es-2.4.6 ✗ ansible-lint playbooks/*.yml
[206] Variables should have spaces before and after: {{ var_name }}
/home/pau/dev/timeoverflow-provisioning/roles/elasticsearch/tasks/main.yml:13
  template: src=elasticsearch.j2 dest={{instance_sysd_script}} mode=0644 force=yes
```

Reply:
> but not all

I see, but I don't understand :see_no_evil: It's a bug? I any case, go ahead since "more is better"...